llvm/lib/MC/MCObjectFileInfo.cpp

```diff
@@ -1172,7 +1172,7 @@ MCObjectFileInfo::getCallGraphSection(const MCSection &TextSec) const {
   }
 
   return Ctx->getELFSection(
-      ".llvm.callgraph", ELF::SHT_PROGBITS, Flags, 0, GroupName,
+      ".llvm.callgraph", ELF::SHT_LLVM_CALL_GRAPH_SECTION, Flags, 0, GroupName,
```

Comment:
```suggestion
      ".llvm.callgraph", ELF::SHT_LLVM_CALL_GRAPH, Flags, 0, GroupName,
```

I'd suppose this would be the more typical naming pattern based on other SHT_LLVM* types I see in tree.

Reply:
There is a `SHT_LLVM_CALL_GRAPH_PROFILE` that already exists. Used _SECTION to distinguish between the two types. But not opposed to dropping _SECTION suffix.